src/Content/googleTutorialContent.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
I'd consider keying this by domain or something (like just "google.com") so when the page's domain is extracted from the URL bar or current tab, you can just access it in the map via something like `let currentTutorial = tutorialsMap[currentDomain]`

Reply:
So the first line I should insert the key "domain" : "google.com" Then edit it so all the pop up keys are inside of it?